src/desktop/apps/artwork/helper.js

```diff
@@ -1,15 +1,42 @@
-import { pickBy, identity } from "lodash"
+import { identity, map, pickBy, trim } from "lodash"
+import { toSentence } from "underscore.string"
 const { APP_URL } = require("sharify").data
 
+const AVAILABILITY = {
+  "for sale": "https://schema.org/InStock",
```

Comment:
are we sure these two are the only options for `availability`?

Reply:
It's for sure not the only one we support but not quite sure how we would reflect those in a way that matches the schema https://schema.org/ItemAvailability

Reply:
Just make sure you have a fallback for states that don't map in this object. You can have any of these types based on schema.org. https://schema.org/ItemAvailability

- Discontinued
- InStock
- InStoreOnly
- LimitedAvailability
- OnlineOnly
- OutOfStock
- PreOrder
- PreSale
- SoldOut

Reply:
I talked to @junybrum and she mentioned that we wouldn't want to display the availability for artworks that aren't either `for_sale` or `sold`.